loader/loader.c

```diff
@@ -67,6 +67,9 @@
 #if defined(WIN32)
 #include "loader_windows.h"
 #endif
+#if defined(__linux__) || defined(__QNXNTO__) || defined(__FreeBSD__)
```

Comment:
Wouldn't be bad to have a single definition of the `linux | qnx | freebsd` so its not repeated everywhere.